### PR TITLE
Implement general Chrono Minor behaviour

### LINKED ISSUE #31: Chrono Miners should warp back to refineries
When returning to a refinery, a Chrono Miner should teleport if a spot is available. If none is, I forget the exact behaviour; IIRC it would drive back until a spot opened up.

Revision: 41d9a20

### PR DESCRIPTION
**Pull request by ghost**
The Chrono Miner now teleports back to the refinery, It might be wise to
improve upon its refinery selection process, but this is the most
important thing.

Fixes #31

### CODE REVIEW

File: mods/ra2/mod.yaml [hunk 1]
```diff
@@ -98,6 +98,7 @@ Assemblies:
 	./mods/common/OpenRA.Mods.Common.dll
 	./mods/ra/OpenRA.Mods.RA.dll
 	./mods/ts/OpenRA.Mods.TS.dll
+    ./mods/ra2/OpenRA.Mods.RA2.dll
```
Comment (MustaphaTR) [inline mods/ra2/mod.yaml:101]: `spaces` to `tab`

File: OpenRA.Mods.RA2/Activities/ChronoDeliverResources.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.RA2.Activities
+{
+	public class ChronoDeliverResources : DeliverResources
+	{
+		public ChronoDeliverResources(Actor self)
+			: base (self)
+		{ }
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/Activities/ChronoDeliverResources.cs:12]: `: base(self) { }`, please.

File: OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj [hunk 3]
```diff
@@ -2,9 +2,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj:5]: Those changes seem unrelated?

File: OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj [hunk 4]
```diff
@@ -46,6 +44,8 @@
     </CustomCommands>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <WarningLevel>4</WarningLevel>
+    <Optimize>false</Optimize>
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj:48]: Same here.

File: OpenRA.Mods.RA2/Traits/ChronoHarvester.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	public class ChronoHarvesterInfo : HarvesterInfo
+	{
+		public override object Create(ActorInitializer init) { return new ChronoHarvester(init.Self, this); }
+	}
+
+	public class ChronoHarvester : Harvester
+	{
+		public ChronoHarvester(Actor self, ChronoHarvesterInfo info)
+			: base (self, info)
+		{
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/Traits/ChronoHarvester.cs:18]: `: base(self, info) { }`

File: OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj (hunk 4 above)
Comment (GraionDilach) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj:48]: These changes look something made by the programming tool (MonoDevelop?)

Comment (phrohdoh) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj:48]: VS/XS/MD make these changes, but we do not want them.
@scshunt please only commit file additions/removals from csproj files.

**Comment by GraionDilach** [comment]
Why was this deleted? It certainly was a step towards the right direction. O_o

**Comment by phrohdoh** [comment]
In favor of a better implementation.
#53
